google-cloud-firestore/src/test/java/com/google/cloud/firestore/it/ITE2ETracingTest.java

```diff
@@ -319,20 +319,13 @@ public void before() throws Exception {
     // Initialize the Firestore DB w/ the OTel SDK. Ideally we'd do this is the @BeforeAll method
     // but because gRPC traces need to be deterministically force-flushed, firestore.shutdown()
     // must be called in @After for each test.
-    FirestoreOptions.Builder optionsBuilder;
-    if (isUsingGlobalOpenTelemetrySDK()) {
-      optionsBuilder =
-          FirestoreOptions.newBuilder()
-              .setOpenTelemetryOptions(
-                  FirestoreOpenTelemetryOptions.newBuilder().setTracingEnabled(true).build());
-    } else {
+    FirestoreOptions.Builder optionsBuilder = FirestoreOptions.newBuilder();
```

Comment:
Question, I saw there are 4 classes like `ITTracingTestGlobalOtel extends ITTracingTest `, `ITTracingTestNonGlobalOtel extends ITTracingTest`, but they are not used anywhere.  Plus, the default behaviour is being changed to "use custom otel or fall back to Global otel", do we still need that 4 classes?

Reply:
Regarding `they are not used anywhere`:

`ITTracingTest` and `ITE2ETracingTest` are abstract base classes, and therefore they actually won't run any tests. When the `it` test suite is run, it will run the inherited classes (`ITE2ETracingTestGlobalOtel`, `ITE2ETracingTestNonGlobalOtel`, `ITTracingTestGlobalOtel`, and `ITTracingTestNonGlobalOtel`). So there are 2 test suites, each one runs in two different configurations (a total of 4). The first two use in-memory trace exporter, and the last two use Cloud Trace backend.

Reply:
Regarding `the default behaviour is being changed to "use custom otel or fall back to Global otel", do we still need that 4 test classes?`

We still need all these tests.

`ITE2ETracingTestGlobalOtel` and `ITTracingTestGlobalOtel` ensure that we collect traces when a global OTEL is registered, _but not passed to Firestore_.

`ITE2ETracingTestNonGlobalOtel` and `ITTracingTestNonGlobalOtel` ensure that we collect traces using the OTEL instance that has been passed to Firestore.